agent/util-scripts/pbench-tool-meister-start

```diff
@@ -646,6 +655,14 @@ def main(_prog: str, cli_params: Namespace) -> int:
         logger.error("invalid sysinfo option(s), '{}'", ",".join(bad_l))
         return ReturnCode.BADSYSINFO
 
+    if cli_params.orchestrate not in _orchestrate_choices:
+        logger.error(
+            "invalid --orchestrate directive, '%s', expected one of %s",
+            cli_params.orchestrate,
+            ", ".join(_orchestrate_choices),
+        )
+        return ReturnCode.INVALIDORCHESTRATE
```

Comment:
I wonder if this is because the "default" is immune to choice validation? And, if so, whether Click's environment variable support would avoid the need for this?

I didn't say that out loud, either ... did I?

Reply:
I've been resisting moving anything related to the CLI interfaces to the Tool Meister to `click` because I don't want it to maintain official CLI interfaces to the Tool Meister just yet.

I'll file a tracking issue for this, none the less.

Reply:
Tracked in #2316.

Reply:
> I've been resisting moving anything related to the CLI interfaces to the Tool Meister to `click` because I don't want it to maintain official CLI interfaces to the Tool Meister just yet.

Indeed; which is why I didn't say any of that. But tracking it with an issue is good.